README.md

```diff
@@ -1,28 +1,217 @@
-      * ODOO_CONF_db_user=odoo
-      * ODOO_CONF_limit_memory_soft=671088640
-    * then, in the boot script:
-      * loop over each parameter of odoo.conf
-      * check if there's an ENV variable "ODOO_CONF_" + parameter
-      * if so, replace the parameter in odoo.conf
-  2. Other suggestion:
-    * remove all optional parameters from odoo.conf template, only keep the mandatory/important
-      ones (e.g. admin_passwd, db_user, etc.)
-    * replace those parameters using the current function (or using solution #1)
-    * either:
-      1. append a user defined odoo.conf to the one in the image (add this user defined
-      odoo.conf by Dockerfile or by volume)
-      2. or use ENV variables to customize parameters (I'm more in favor of that solution)
\ No newline at end of file
+# elicocorp/odoo
+Simple yet powerful Odoo image for Docker based on [OCB][ocb] code and
+maintained by [Elico Corp][ec].
+
+  [ocb]: https://github.com/OCA/OCB "Odoo Community Backports"
+  [ec]: https://www.elico-corp.com/
+
+This image is a fork of [XCG Consulting][xcg] Odoo Docker image available
+[here][xcgd].
+
+  [xcg]: https://www.xcg-consulting.fr/
+  [xcgd]: https://hub.docker.com/r/xcgd/odoo/
+
+## Usage
+In order to use this image, a recent version of [Docker][dk] must be installed
+on the target host. For more information about Docker Engine, see the
+[documentation][dk-doc].
+
+  [dk]: https://www.docker.com/
+  [dk-doc]: https://docs.docker.com/engine/
+
+Running this image without specifying a command will display this help message:
+
+    docker run elicocorp/odoo:10.0
+
+The easiest way to use this image is to run it along with a [PostgreSQL][pg]
+image.
+
+  [pg]: https://hub.docker.com/_/postgres/
+
+**Note:** The [PostgreSQL image][ec-pg] of Elico Corp can be used as well.
+
+  [ec-pg]: https://hub.docker.com/r/elicocorp/postgres/
+
+Below is an example of a simple `docker-compose.yml` to use this image.
+For more information about Docker Compose, see the [documentation][dc-doc].
+
+  [dc-doc]: https://docs.docker.com/compose/
+
+    version: '3.3'
+    services:
+
+      postgres:
+        image: postgres:9.5
+        environment:
+          - POSTGRES_USER=odoo
+        network_mode: bridge
+
+      odoo:
+        image: elicocorp/odoo:10.0
+        command: start
+        ports:
+         - 127.0.0.1:8069:8069
+        links:
+         - postgres:db
+        environment:
+         - ODOO_DB_USER=odoo
+        network_mode: bridge
+
+**Note 1:** With this configuration, Odoo will be accessible at the following
+URL *only* from the local host: <http://127.0.0.1:8069>
+
+It is possible to publish it following one of these options:
+
+1. map a reverse-proxy to `127.0.0.1:8069`
+2. remove `127.0.0.1:` in order to publish the port `8069`
+
+**Note 2:** With this configuration:
+
+1. Odoo is running without master password
+2. `odoo` PostgreSQL user is a superuser who doesn't require any password
+
+See [Security] section for more info.
```

Comment:
Strange, this doesn't seem to be working though [it was supposed to be implemented](https://github.com/jgm/pandoc/commit/c435e9cda7eb73cf4d8c446bb829079dddfaedbc) in MD...